src/System Application/App/File System/permissions/FileSystemObjects.PermissionSetExt.al

```diff
@@ -0,0 +1,34 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.FileSystem;
+
+permissionset 9452 "File System - Objects"
+{
+    Access = Internal;
+    Assignable = false;
+
+    Permissions =
+        table "File Account" = X,
```

Comment:
consider adding InherentPermissions for the objects everyone should be able to always execute.